common/helper.py

```diff
@@ -649,7 +649,6 @@ def cmd_exec(cmd, env=None, cwd=None, shell=True, log=None, verbose=True, hide=N
                                            check=True,
                                            stdout=subprocess.PIPE,
                                            stderr=subprocess.STDOUT,
-                                           encoding='utf-8',
```

Comment:
It's not quite good idea to disable encoding.
Need to be shure that output has correct format.

Reply:
I agree. Should be added default parameter in this case.